lib/suspenders/generators/app_generator.rb

```diff
@@ -138,18 +143,16 @@ def remove_routes_comment_lines
     end
 
     def generate_default
-      run("spring stop > /dev/null 2>&1")
+      run("spring stop > /dev/null 2>&1 || true")
       generate("suspenders:runner")
       generate("suspenders:profiler")
       generate("suspenders:json")
       generate("suspenders:static")
-      generate("suspenders:stylesheet_base")
+      generate("suspenders:stylesheet_base") unless options[:api]
```

Comment:
This prevents `api_spec.rb` from failing because it tries to run `yarn` but there's no such command in API mode. I'm sure there are other generators we'll want to skip in that case, and we'll probably want to address them in a future PR.